### PR TITLE
build: refactor releasing process

### PR DESCRIPTION
**Pull request by troian**
- add akash-edge homebrew
- extend goreleaser build flags via environment variables

Signed-off-by: Artur Troian <troian.ap@gmail.com>

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -10,3 +10,4 @@ AKASH_DEVCACHE_VERSIONS=${AKASH_DEVCACHE}/versions
 AKASH_DEVCACHE_NODE_MODULES=${AKASH_DEVCACHE}
 AKASH_DEVCACHE_NODE_BIN=${AKASH_DEVCACHE_NODE_MODULES}/node_modules/.bin
 AKASH_RUN=${AKASH_DEVCACHE}/run
+ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.18
```
Comment (hydrogen18) [inline .env:13]: What is this doing? Disabling some sort of check around pointers?

File: Makefile [hunk 2]
```diff
@@ -26,55 +26,51 @@ GIT_HEAD_COMMIT_LONG  := $(shell git log -1 --format='%H')
...
-	GORELEASER_HOMEBREW_NAME=akash
-	GORELEASER_HOMEBREW_CUSTOM=
+RELEASE_TAG           ?= $(shell git describe --tags --abbrev=0)
+IS_PREREL             := $(shell $(ROOT_DIR)/script/is_prerelease.sh "$(RELEASE_TAG)")
+IS_MAINNET            := $(shell $(ROOT_DIR)/script/mainnet-from-tag.sh "$(RELEASE_TAG)")
+
+GO_LINKMODE            ?= external
+GO_MOD                 ?= readonly
+BUILD_TAGS             ?= osusergo,netgo,ledger,static_build
+GORELEASER_STRIP_FLAGS ?=
```
Comment (hydrogen18) [inline Makefile:36]: Does this line do anything?

Comment (troian) [reply Makefile:36]: yeah, it is set to `-s -w` if `BUILD_OPTIONS` does not contain `nostrip` (line 66)
in other words it is set to `GORELEASER_STRIP_FLAGS='-w -d'` unless `BUILD_OPTIONS=nostrip` used to call make target

File: .env (hunk 1 above)
Comment (troian) [reply .env:13]: thanks for catching. removing as it was only for testing

**Review by hydrogen18 — APPROVED** [review]

**Pull request merged**